examples/app_unity_gain_voice/Makefile

```diff
@@ -8,15 +8,15 @@ TARGET = XCORE-200-EXPLORER
 # the project name
 APP_NAME = app_unity_gain_voice
 
-# The USED_MODULES variable lists other module used by the application.
-USED_MODULES = lib_src(>=1.1.0)
-
 # The flags passed to xcc when building the application
 # You can also set the following to override flags for a particular language:
 # XCC_XC_FLAGS, XCC_C_FLAGS, XCC_ASM_FLAGS, XCC_CPP_FLAGS
 # If the variable XCC_MAP_FLAGS is set it overrides the flags passed to
 # xcc for the final link (mapping) stage.
-XCC_FLAGS = -O2 -g
+XCC_FLAGS = -O2 -g -DDEBUG_PRINT_ENABLE=0
+
+# The USED_MODULES variable lists other module used by the application.
+USED_MODULES = lib_src(>=1.1.1) lib_logging
```

Comment:
Please could you set a minimum version requirement for the library here. This allows the tools to give warnings when the major version of dependency is incremented (as it may no longer behave as expected), and also a useful build errors if the version used is not high enough.